lib/core/utils/clone.js

```diff
@@ -8,6 +8,10 @@ function clone(obj) {
   var index,
     length,
     out = obj;
+  // DOM nodes cannot be cloned.
+  if (window?.Node && obj instanceof window.Node) {
```

Comment:
In IE11 this fails as a node can also be an `HTMLCollection`.

```suggestion
  if (
    (window?.Node && obj instanceof window.Node) || 
    (window?.HTMLCollection && obj instanceof window.HTMLCollection)
  ) {
```